webapp-django/crashstats/settings/base.py

```diff
@@ -376,12 +346,260 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
+# or if using the debug toolbar, you might give toolbar a try
+STATSD_CLIENT = config('STATSD_CLIENT', 'django_statsd.clients.null')
+
+# for local development these don't matter
+STATSD_HOST = config('STATSD_HOST', 'localhost')
+STATSD_PORT = config('STATSD_PORT', 8125, cast=int)
+STATSD_PREFIX = config('STATSD_PREFIX', None)
+
+# Enable this to be able to run tests
+# NB: Disable this caching mechanism in production environment as
+# it will break work of anonymous CSRF if there is more than one
+# web server thread.
+# Comment out to use memcache from settings/base.py
+CACHES = {
+    'default': {
+        # use django.core.cache.backends.memcached.MemcachedCache for prod
+        'BACKEND': config(
+            'CACHE_BACKEND',
+            'django.core.cache.backends.locmem.LocMemCache',
+        ),
+        'LOCATION': config(
+            'CACHE_LOCATiON',
+            'crashstats',
+        ),
+        'TIMEOUT': config('CACHE_TIMEOUT', 500),
+        'KEY_PREFIX': config('CACHE_KEY_PREFIX', 'crashstats'),
+    }
+}
+
+TIME_ZONE = config('TIME_ZONE', 'UTC')
+
+# FIXME should support URLs here!
+DATABASES = {
+    'default': {
+        # use django.db.backends.postgresql_psycopg for production
+        'ENGINE': config('DATABASE_ENGINE', 'django.db.backends.sqlite3'),
+        'NAME': config('DATABASE_NAME', 'sqlite.crashstats.db'),
+        'USER': config('DATABASE_USER', ''),
+        'PASSWORD': config('DATABASE_PASSWORD', ''),
+        'HOST': config('DATABASE_HOST', ''),
+        'PORT': config('DATABASE_PORT', ''),
+        'OPTIONS': {
+        },
+        'TEST_CHARSET': 'utf8',
+        'TEST_COLLATION': 'utf8_general_ci',
+    },
+    # 'slave': {
+    #     ...
+    # },
+}
+
+
+# Uncomment this and set to all slave DBs in use on the site.
+SLAVE_DATABASES = config('SLAVE_DATABASES', None, cast=Csv)
+
+# TODO not sure how we can override this, should we bother?
+# Recipients of traceback emails and other notifications.
+ADMINS = (
+    # ('Your Name', 'your_email@domain.com'),
+)
+MANAGERS = ADMINS
+
+# Debugging displays nice error messages, but leaks memory. Set this to False
+# on all server instances and True only for development.
+DEBUG = TEMPLATE_DEBUG = config('DEBUG', True, cast=bool)
+
+# Set this to True to make debugging AJAX requests easier; development-only!
+DEBUG_PROPAGATE_EXCEPTIONS = config(
+    'DEBUG_PROPAGATE_EXCEPTIONS',
+    True,
+    cast=bool
+)
+
+# By default compression is done in runtime, if you enable
+# offline compression, running the test suite will be 10 times faster
+# but you'll need to remember to first run:
+#     ./manage.py collectstatic --noinput
+#     ./manage.py compress --force --engine=jinja2
+# at least once every time any of the static files change.
+COMPRESS_OFFLINE = config('COMPRESS_OFFLINE', True, cast=bool)
+
+# Make this unique, and don't share it with anybody.  It cannot be blank.
+SECRET_KEY = config('SECRET_KEY', 'you must change this')
```

Comment:
If you use `SECRET_KEY = config('SECRET_KEY')` I think it'll raise a [UndefinedValueError](https://github.com/henriquebastos/python-decouple/blob/master/decouple.py#L64) if it's not provided with a value. 

[Here's how sugardough handles it](https://github.com/mozilla/sugardough/blob/master/%7B%7B%20cookiecutter.project_name%20%7D%7D/%7B%7B%20cookiecutter.project_name%20%7D%7D/settings/base.py#L24)